.github/workflows/build.yml

```diff
@@ -48,6 +49,30 @@ jobs:
         run: |
           make format || true
           make test
+
+  test-freebsd-amd64:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+      - name: Test
+        uses: vmactions/freebsd-vm@v0
+        with:
+          usesh: true
+          prepare: |
+            wget https://go.dev/dl/go1.20.4.freebsd-amd64.tar.gz
+            tar -C /usr/local -xzf go1.20.4.freebsd-amd64.tar.gz
```

Comment:
Could we interpolate the version somehow so that we're using the same go version across all 3 OSes?